profiles/base.yaml

```diff
@@ -437,9 +437,8 @@ forging_tools:
 - tong
 - bellow
 - shovel
-knitting_tools:
```

Comment:
❌ If you're going to remove it here, please also remove it from base-empty: https://github.com/rpherbig/dr-scripts/blob/fc402777aac0c7bf8bb0ecf431293e5b54fcc227/profiles/base-empty.yaml#L28

❓ Speaking of which, I noticed `outfitting_tools` isn't defined in base-empty, could you add it?